packages/sakuli-cli/src/commands/enable-enterprise/tasks/license-tasks.function.ts

```diff
@@ -3,32 +3,32 @@ import execa from 'execa';
 import { join } from "path";
 import { homedir } from "os";
 
-const LicenseKeyEnv = (key:string) => `SAKULI_LICENSE_KEY=${key}`
+const LicenseKeyEnv = (key:string) => `SAKULI_LICENSE_KEY=${key}`;
 
 /**
  * Task to set SAKULI_LICENSE_KEY env var on Windows systems
  * @param key
  */
 const licenseGlobalWinTask = (key: string) => async () => {
     await execa('setx', [LicenseKeyEnv(key)]);
-}
+};
 
 /**
  * Task to set SAKULI_LICENSE_KEY env var on *Nix systems and Mac
  * @param key
  */
 const licenseGlobalNixTask = (key: string) => async () => {
     const bashRcFile = join(homedir(), '.bashrc');
-    await fs.writeFile(bashRcFile, `\nexports ${LicenseKeyEnv(key)}\n`, {flag: 'a'});
-}
+    await fs.writeFile(bashRcFile, `\nexport ${LicenseKeyEnv(key)}\n`, {flag: 'a'});
+};
 
 /**
  * Creates a task that sets the SAKULI_LICENSE_KEY on the specific platform
- * @param
+ * @param key The license key to be set
```

Comment:
```suggestion
 * @param key - The license key to be set
```

I think the "more correct" format here is to but the `-` between parameter name and description (even if the most UI's also interpret the other notation very well)